src/html/index.html

```diff
@@ -0,0 +1,223 @@
+                <header class="article__add-info">
+                    <p><time datetime="2020-03-10">Mar 10, 2020</time> · 4 min read</p>
+                </header>
+            </article>
+            <article class="articles__article article">
+                <img class="article__image" src="assets/images/Image3.png" alt="imageOfArticle3">
+                <a class="article__link" href="#">
+                    <h4 class="article__header">Improve your customer experience</h4>
+                </a>
+                <p class="article__description">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+                    Sint harum rerum voluptatem quo recusandae magni placeat saepe molestiae, sed excepturi cumque corporis perferendis hic.
+                </p>
+                <div class="article__spacer-box"></div>
+                <header class="article__add-info">
+                    <p><time class="article__time" datetime="2020-02-12">Feb 12, 2020</time> · 11 min read</p>
+                </header>
+            </article>
+        </section>
+        <section class="main__add-article add-article">
+            <form class="add-article__form add-article__form--disabled">
+                <label class="add-article__label">Название статьи</label>
+                <input class="add-article__field add-article__name" type="text" name="articleName">
+                <label class="add-article__label">Ссылка на картинку</label>
+                <input class="add-article__field add-article__link" type="text" name="articleLink">
+                <label class="add-article__label">Описание</label>
+                <textarea class="add-article__field add-article__description" type="text" name="articleDescription"></textarea>
+                <button class="add-article__btn-cancel" name="button" type="button">Отмена</button>
+                <button class="add-article__btn-create" name="button" type="button">Создать</button>
+            </form>
+            <button class="add-article__btn-add">Добавить</btn>
+        </section>
+        <section class="main__faq faq">
+            <header class="faq__heading">
+                <h3 class="faq__header">FAQ</h3>
+            </header>
+            <dl class="faq__dl">
+                <dt class="faq__dt dt">
+                    <button class="dt__text">Название вопроса 1</button>
+                    <button class="dt__btn">
+                        <svg class="faq__svg faq__svg--down" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M12 19L12 5" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                            <path d="M5 12L12 5L19 12" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                        </svg>
+                    </button>
+                </dt>
```

Comment:
тут правильней всего сделать 1 button
```suggestion
                <dt class="faq__dt dt">
                    <button class="dt__text">
                        <span>Название вопроса 1</span>
                        <svg class="faq__svg faq__svg--down" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M12 19L12 5" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
                            <path d="M5 12L12 5L19 12" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
                        </svg>
                    </button>
                </dt>
```

Reply:
т.е при нажатии на название (на весь блок), должна меняться стрелка, у вас сейчас если только нажать на стрелку меняется стрелка.

`<button >`  где будет текст и иконка сделать блочным